import_precheck/check_osversion.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2017 Google Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/osinfo"
+	"github.com/crunk1/golang-sak"
+)
+
+var OSVERSIONS = map[string][]string{
+	"centos":  {"6", "7"},
+	"debian":  {"8", "9"},
+	"ol":      {"6", "7"},
+	"rhel":    {"6", "7"},
+	"ubuntu":  {"14.04", "16.04"},
+	"windows": {"6.1", "6.3", "10.0"},
+}
+
+type OSVersionCheck struct{}
+
+func (c *OSVersionCheck) GetName() string {
+	return "OS Version Check"
+}
+
+func (c *OSVersionCheck) Run() (*Report, error) {
+	r := &Report{Name: c.GetName()}
+	var versions []string
+	var ok bool
+	info, err := osinfo.GetDistributionInfo()
+	if err != nil {
+		return nil, err
+	}
+	r.AddNote(fmt.Sprintf("OSInfo: %+v", *info))
+
+	if versions, ok = OSVERSIONS[info.ShortName]; !ok {
+		r.AddIncompatibility(info.ShortName)
+		return r, nil
+	}
+
+	contains := func(i interface{}) interface{} {
+		if strings.Contains(info.Version, i.(string)) {
+			return i
+		}
+		return nil
+	}
+	if version := sak.Any(versions, contains); version == nil {
```

Comment:
This function is super simple, just reimplement instead of pulling in a thirdparty source